cmd/podman/rm.go

```diff
@@ -195,5 +195,10 @@ func rmCmd(c *cliconfig.RmValues) error {
 			exitCode = 1
 		}
 	}
+
+	if failureCnt > 0 {
```

Comment:
This is the default, why are you setting it?
If an err is returned and failureCnt == 0 then it is 1,
else if err is returned exitCode=125

Reply:
Ok so this fixed the issue where you have a non existing container and one of the other containers causes an error.
Could you add a test for this situation.

Reply:
This is a continuation of commit 85db895012bead6b which was meant for #2539.

Getting the list of containers and deleting them are two different steps which can throw different kinds of errors. I guess `failureCnt` is meant to keep track of errors that aren't `libpod.ErrNoSuchCtr`, right? For the case of a bogus and a running container, the first step will throw `libpod.ErrNoSuchCtr`, and we set `exitCode` to 1 and `failureCnt` stays 0. Then later when we hit another error due to the state of the container, `failureCnt` is incremented but `exitCode` doesn't get reset.

I have now added this text to the commit message also.

Reply:
Yes, the branch has a test. :)